src/coreComponents/fileIO/schema/schema.xsd

```diff
@@ -594,19 +589,44 @@ Available options are: jacobi, blockJacobi, gaussSeidel, blockGaussSeidel, cheby
 		<!--krylovTol => Relative convergence tolerance of the iterative method
 If the method converges, the iterative solution :math:`\mathsf{x}_k` is such that
 the relative residual norm satisfies:
-:math:`|| \mathsf{b} - `\mathsf{A} \mathsf{x}_k ||_2` < ``tolerance`` * :math:`|| \mathsf{b} ||_2`|-->
+:math:`\left\lVert \mathsf{b} - \mathsf{A} \mathsf{x}_k \right\rVert_2` < ``krylovTol`` * :math:`\left\lVert\mathsf{b}\right\rVert_2`-->
 		<xsd:attribute name="krylovTol" type="real64" default="1e-06" />
 		<!--krylovWeakestTol => Weakest-allowed tolerance for adaptive method-->
 		<xsd:attribute name="krylovWeakestTol" type="real64" default="0.001" />
 		<!--logLevel => Log level-->
 		<xsd:attribute name="logLevel" type="integer" default="0" />
-		<!--preconditionerType => Preconditioner type
-Available options are: none, jacobi, iluk, ilut, icc, amg, mgr, block-->
-		<xsd:attribute name="preconditionerType" type="string" default="iluk" />
-		<!--solverType => Linear solver type
-Available options are: direct, cg, gmres, fgmres, bicgstab, preconditioner-->
-		<xsd:attribute name="solverType" type="string" default="direct" />
-	</xsd:complexType>
+		<!--preconditionerType => Preconditioner type. Available options are:
+* none
+* jacobi
+* gs
+* sgs
+* iluk
+* ilut
+* icc
+* ict
+* amg
+* mgr
+* block-->
+		<xsd:attribute name="preconditionerType" type="geosx_LinearSolverParameters_PreconditionerType" default="iluk" />
+		<!--solverType => Linear solver type. Available options are:
+* direct
+* cg
+* gmres
+* fgmres
+* bicgstab
+* preconditioner-->
+		<xsd:attribute name="solverType" type="geosx_LinearSolverParameters_SolverType" default="direct" />
+	</xsd:complexType>
+	<xsd:simpleType name="geosx_LinearSolverParameters_PreconditionerType">
+		<xsd:restriction base="xsd:string">
+			<xsd:pattern value=".*[\[\]`$].*|none|jacobi|gs|sgs|iluk|ilut|icc|ict|amg|mgr|block" />
```

Comment:
These would be better served by `xsd:enumeration` than a regexp, but this would require more substantial changes to our schema generation code